docs/internal-documentation/templates/specifications/tests/Template01.md

```diff
@@ -223,17 +228,17 @@ respected.
 
 
 > > ----
-> > The text below is invisible when rendered by Github.
+> > The links listed below are not invisible when rendered by Github.
```

Comment:
"are not invisible" -> "are not visible"

Reply:
I thought I corrected it last round, but I obviously incorrected it. :-)

Corrected correctly now.